library/src/test/java/com/schibsted/spain/barista/rule/flaky/FlakyStatementBuilderTest.java

```diff
@@ -44,6 +44,16 @@ public void repeatStatementReturnedWhenRepeatAnnotationFound() throws Exception
     assertTrue(resultStatement instanceof RepeatStatement);
   }
 
+  @Test
+  public void repeatStatementReturnedWhenRepeatSetRepeatAttemptsByDefault() throws Exception {
```

Comment:
Sorry, I don't get the `repeat set repeat` 😅 

Maybe `repeatStatementReturnedWhenSettingDefaultRepeatAttempts`?

Reply:
Yep, it make sense 👍 , changed